stellargraph/layer/__init__.py

```diff
@@ -36,3 +36,4 @@
 from .rgcn import *
 from .watch_your_step import *
 from .knowledge_graph import *
+from .gcn_lstm import *
```

Comment:
Both `gcn` and `gcn_lstm` now include a class called `GraphConvolution`, which means these `*` imports will shadow each other as the `stellargraph.layer.GraphConvolution` name. I think in particular the `gcn_lstm` one will "win" so existing code importing `stellargraph.layer.GraphConvolution` will switch from using GCN one to using GCN-LSTM one. They're not interchangeable so that code will break.

One way to fix this would be to rename the `gcn_lstm` `GraphConvolution` to something like `FixedAdjacencyGraphConvolution` or even just `LSTMGraphConvolution` so that it no longer overlaps with the `gcn` one.

Reply:
I didn't think about it causing conflicts. I'll fix the name. 
However, why would the `gcn_lstm`'s `GraphConvolution` would win  in the import?